config.py.in

```diff
@@ -1,10 +1,13 @@
+import os
+
 # Server Specific Configurations
 server = {
-    'port': 8081,
-    'host': '0.0.0.0'
+    'port': os.environ.get('PULPITO_SERVER_PORT', '8081'),
+    'host': os.environ.get('PULPITO_SERVER_HOST', '0.0.0.0')
 }
 
-paddles_address = 'http://sentry.front.sepia.ceph.com:8080'
+paddles_address = os.environ.get('PULPITO_PADDLES_ADDRESS',
+                                 'http://sentry.front.sepia.ceph.com:8080')
```

Comment:
This breaks the 'pulpito' role in ceph-cm-ansible: https://github.com/ceph/ceph-cm-ansible/blob/master/roles/pulpito/tasks/setup_pulpito.yml#L51-56

Reply:
Corresponding PR #71